packages/pilot/src/containers/RecipientDetails/Config/index.js

```diff
@@ -67,6 +65,30 @@ class RecipientDetailConfig extends Component {
     })
   }
 
+  onCancel () {
+    this.setState({
+      anticipation: this.props.anticipation,
+      transfer: this.props.transfer,
+      bankAccount: this.props.bankAccount,
+      expanded: {},
+    })
+  }
+
+  onSaveAnticipation (data) {
```

Comment:
`onSaveAnticipation`, `onSaveTransfer` e `onSaveBankAccount` tem um comportamento parecido, poderia existir uma função que recebe a função a ser executada como parametro e chama ela com o data, para não precisar tres funções muito semelhantes. Seria mais ou menos isso aqui:
```suggestion
 handleSave (saveFunction) {
    return data => saveFunction(data)
      .then(() => this.setState({ expanded: {} }))
  }
```

e a chamada ficaria assim:
```js
onSave={this.handleSave(onSaveAnticipation)}
```

Reply:
Feito!

Reply:
@rsmelo Ao testar o `handleSave` que nós alteramos no _Storybook_ obtivemos alguns problemas. O mesmo quebra obrigando a gente a passar uma `Promise.resolve` como mock.

Eu estava fazendo isso para mantermos o padrão que estávamos seguindo para termos a nossa `action` funcionando conforme, porém isso não acontecia.

Conversando com @tisoap voltamos a dicussão :smile: E com isso a gente volta a ter três funções _ save_ para cada um. Por mais que elas possa ser parecidas (e são) cada uma trata um objeto diferente. Por conta disso decidimos voltar ela como estava.